File: 3542MinimumOperationstoConvertAllElementstoZero/MinimumOperationstoConvertAllElementstoZero.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minOperations = function(nums) {
    let count=0;
    let stack=[-1];
    nums.forEach(num=>
         {
            while(num<stack[stack.length-1]){
                stack.pop()
            }
            if(num>stack[stack.l-1]){
                count+=num>0?1:0;
                stack.push(num)
            }
         }

    )

    return count

};

File: 3542MinimumOperationstoConvertAllElementstoZero/MinimumOperationstoConvertAllElementstoZero.java
import java.util.LinkedList;
import java.util.List;

class MinimumOperationstoConvertAllElementstoZero{

    class Solution{
       public int minOperations(int[] nums) {
           int count=0;
           List<Integer> stack=new LinkedList<>();
           stack.add(-1);
           for(int num:nums){
                 while(num<stack.get(stack.size()-1)){
                     stack.remove(stack.size()-1);
                 }
                 if(num>stack.get(stack.size()-1)){
                    count+=num>0?1:0;
                    stack.add(num);

                 }
           }


           return count;
       }
    }
    public static void main(String[] args) {
        
    }
}